xbmc/video/VideoDatabase.cpp

```diff
@@ -4627,10 +4627,9 @@ bool CVideoDatabase::GetPlayCounts(const std::string &strPath, CFileItemList &it
   return false;
 }
 
-int CVideoDatabase::GetPlayCount(const CFileItem &item)
+int CVideoDatabase::GetPlayCount(int iFileId)
```

Comment:
No. The diff is misleading. I did not replace GetPlayCount, but kept the existing 
https://github.com/xbmc/xbmc/pull/6207/files#diff-ee6ba71d6c3b070dfdba880abc06aeaeL409
and introduced two new
https://github.com/xbmc/xbmc/pull/6207/files#diff-ee6ba71d6c3b070dfdba880abc06aeaeR416 and https://github.com/xbmc/xbmc/pull/6207/files#diff-ee6ba71d6c3b070dfdba880abc06aeaeR880. The latter is private and now provides the actual implementation. The two others (reimplementation of existing method as well as the public new one) both use the private method. See here https://github.com/xbmc/xbmc/pull/6207/files#diff-dbf978b6cac348a2af93d3beec420e43R4659 and https://github.com/xbmc/xbmc/pull/6207/files#diff-dbf978b6cac348a2af93d3beec420e43R4664